collateral-onboarding/collateral-onboarding/Dockerfile

```
@@ -0,0 +1,50 @@
+FROM ubuntu:latest as base-installs
+
+RUN apt-get update && \
+    apt-get -y install curl xz-utils git make jq python3 pip
+RUN pip install requests
+
+FROM base-installs as tool-installs
+
+RUN useradd -s /bin/bash -d /home/blockchain/ -m -G sudo blockchain
+RUN mkdir -m 0755 /nix && chown blockchain /nix
+USER blockchain
+ENV USER="blockchain"
+ENV HOME="/home/${USER}"
+ENV PATH="${HOME}/.nix-profile/bin:${PATH}"
+ENV NIX_PATH="${HOME}/.nix-defexpr/channels"
+RUN curl -L https://nixos.org/nix/install | sh -s -- --no-daemon
+RUN echo kek
+RUN curl -L https://dapp.tools/install | sh
+
```

Would this also run without errors?

```suggestion
RUN curl -L https://dapp.tools/install | sh
RUN seth --version
```

---

```
 => ERROR [tool-installs 6/6] RUN seth --version                                                                                                                                                                                                                                                                         0.4s
------
 > [tool-installs 6/6] RUN seth --version:
#12 0.215 /bin/sh: 1: seth: not found
------
executor failed running [/bin/sh -c seth --version]: exit code: 127

---

Thanks! From this I assume that `dapp` tools did not correctly install despite finishing without error code. Correct me if I am missing something here